templates/default/sql/dml/0.10.0.sql.erb

```diff
@@ -30,3 +30,9 @@ REPLACE INTO `hopsworks`.`variables` VALUES ("kagent_liveness_threshold", "<%= n
 REPLACE INTO `hopsworks`.`variables` VALUES ("service_restart_script", "<%= node['hopsworks']['kagent_liveness']['restart_script'] %>");
 REPLACE INTO `hopsworks`.`variables` VALUES ("service_start_script", "<%= node['hopsworks']['kagent_liveness']['start_script'] %>");
 REPLACE INTO `hopsworks`.`variables` VALUES ("service_stop_script", "<%= node['hopsworks']['kagent_liveness']['stop_script'] %>");
+
+
+REPLACE INTO `hopsworks`.`variables` VALUES ("sklearn_serving_connection_pool_size", "<%= node['sklearn_serving']['pool_size'] %>");
+REPLACE INTO `hopsworks`.`variables` VALUES ("sklearn_serving_max_route_connections", "<%= node['sklearn_serving']['max_route_connections'] %>");
+
+REPLACE INTO `hopsworks`.`schema_topics`(`name`, `version`, `contents`) VALUES ('inferenceschema', 1, '{"fields": [{"name": "modelId", "type": "int"}, { "name": "modelName", "type": "string" }, {  "name": "modelVersion",  "type": "int" }, {  "name": "requestTimestamp",  "type": "long" }, {  "name": "responseHttpCode",  "type": "int" }, {  "name": "inferenceRequest",  "type": "string" }, {  "name": "inferenceResponse",  "type": "string" }, { "name": "servingType", "type": "string" } ],  "name": "inferencelog",  "type": "record" }');
```

Comment:
We should bump the version, otherwise it will break current installations, as the schema is different.

Reply:
Done